README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple classes in the same file, each class should have its own file. 
+
+The following list should be the standard struct/class organization of all your Swift files, in this specific order:
+
+* Static variables
+* View Lifecycle (if available)
```

Comment:
`Override` encompasses `viewDidLoad`, `viewWillAppear`, etc, just less explicitly and more generally applies to _any_ class file. I don't like having View lifecycle as a part of this. WE can enforce compliance for that